File: core/src/main/java/com/andersmurphy/geoquiz/presenters/QuizPresenter.java
package com.andersmurphy.geoquiz.presenters;

/**
 * Created by anders on 15/12/14.
 */
public interface QuizPresenter {
    void prepareFirstQuestion();

    void truePressed();

    void falsePressed();

    void nextPressed();

    void previousPressed();

    void saveState();

    void cheatPressed();

    void resultFromCheatScreen(boolean wasAnswerShown);
}

File: core/src/main/java/com/andersmurphy/geoquiz/presenters/QuizPresenter.kt
package com.andersmurphy.geoquiz.presenters

/**
 * Created by anders on 15/12/14.
 */
interface QuizPresenter {
    fun prepareFirstQuestion()
    fun truePressed()
    fun falsePressed()
    fun nextPressed()
    fun previousPressed()
    fun saveState()
    fun cheatPressed()
    fun resultFromCheatScreen(wasAnswerShown: Boolean)
}
